plugins/woocommerce/changelog/add-85

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Implement experiment within code for new users#85
```

Comment:
I think we should remove the `#85` from the changelog entry.  Also a bit picky, but might be good to be more specific with this entry:

```suggestion
Implement the product blocks experiment within code for new users
```